js/html/Index.svelte

```diff
@@ -1,95 +1,91 @@
+<script context="module" lang="ts">
+	export { default as BaseHTML } from "./shared/HTML.svelte";
+	export { default as BaseExample } from "./Example.svelte";
+</script>
+
 <script lang="ts">
-	import type { Gradio } from "@gradio/utils";
+	import { Gradio } from "@gradio/utils";
 	import HTML from "./shared/HTML.svelte";
 	import { StatusTracker } from "@gradio/statustracker";
-	import type { LoadingStatus } from "@gradio/statustracker";
 	import { Block, BlockLabel } from "@gradio/atoms";
 	import { Code as CodeIcon } from "@gradio/icons";
 	import { css_units } from "@gradio/utils";
+	import type { HTMLProps, HTMLEvents } from "./types.ts";
 
-	export let label = "HTML";
-	export let elem_id = "";
-	export let elem_classes: string[] = [];
-	export let visible: boolean | "hidden" = true;
-	export let value = "";
-	export let props: Record<string, any> = {};
-	export let apply_default_css = true;
-
-	export let html_template = "${value}";
-	export let css_template = "";
-	export let js_on_load: string | null = null;
-
-	let _props: Record<string, any> = { value, label, visible, ...props };
-	$: _props = { ..._props, ...props, value, label, visible };
+	let props = $props();
+	const gradio = new Gradio<HTMLEvents, HTMLProps>(props);
 
-	export let loading_status: LoadingStatus;
-	export let gradio: Gradio<{
-		change: never;
-		click: never;
-		submit: never;
-		clear_status: LoadingStatus;
-	}>;
-	export let show_label = false;
-	export let min_height: number | undefined = undefined;
-	export let max_height: number | undefined = undefined;
-	export let container = false;
-	export let padding = true;
-	export let autoscroll = false;
+	let _props = $derived({
+		value: gradio.props.value || "",
+		label: gradio.shared.label,
+		visible: gradio.shared.visible,
+		...gradio.props.props
+	});
 
-	let old_value = value;
-	$: if (JSON.stringify(old_value) !== JSON.stringify(value)) {
-		old_value = value;
-		gradio.dispatch("change");
-	}
+	let old_value = $state(gradio.props.value);
+	$effect(() => {
+		if (JSON.stringify(old_value) !== JSON.stringify(gradio.props.value)) {
```

Comment:
In my experience, svelte 5 is smarter about comparing state objects so I don't think we need the stringify anymore in svelte 5.